sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -585,6 +586,71 @@ public String getId() {
             });
     }
 
+    /**
+     * Executes flux of operations in Bulk.
+     *
+     * @param <TContext> The context for the bulk processing.
+     *
+     * @param operations Flux of operation which will be executed by this container.
+     *
+     * @return A Flux of {@link CosmosBulkOperationRequestAndResponse} which contains operation and it's response or exception.
+     * <p>
+     *     To create a operation which can be executed here, use {@link BulkProcessingUtil}. For eg.
+     *     for a upsert operation use {@link BulkProcessingUtil#getBulkUpsertItemOperation(Object, PartitionKey)}
+     * </p>
+     * <p>
+     *     We can get the corresponding operation using {@link CosmosBulkOperationRequestAndResponse#getOperation()} and
+     *     it's response using {@link CosmosBulkOperationRequestAndResponse#getResponse()}. If the operation was executed
+     *     successfully, the value returned by {@link CosmosBulkItemResponse#isSuccessStatusCode()} will be true. To get
+     *     actual status use {@link CosmosBulkItemResponse#getStatusCode()}.
+     * </p>
+     * To check if the operation had any exception, use {@link CosmosBulkOperationRequestAndResponse#getThrowable()} to
+     * get the exception.
+     */
+    @Beta(Beta.SinceVersion.V4_7_0)
+    public <TContext> Flux<CosmosBulkOperationRequestAndResponse<TContext>> processBulkOperations(
+        Flux<CosmosItemOperation> operations) {
```

Comment:
if the input `operations` Flux sends error signal, what is the expected behaviour?
We should have a test for this scenario to validate the behaviour.

Reply:
Ack. Will do that. Thanks.